OJP/OJP_Trips.xsd

```diff
@@ -162,6 +167,54 @@
 					<xs:documentation>Whether the result should include operating day information - as encoded bit string and in natural language.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="HikingProfile" default="normal" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>Users walking profile (especially for hiking)</xs:documentation>
```

Comment:
Does this apply to every single walking leg (connection, etc.) or just for real hiking legs ?

Reply:
Only for hiking, we have walking speed for the rest I guess. If acceptable, I update it